pkg/cmd/codespace/create.go

```diff
@@ -466,12 +466,18 @@ func getRepoSuggestions(ctx context.Context, apiClient apiClient, partialSearch
 }
 
 // buildDisplayName returns display name to be used in the machine survey prompt.
+// prebuildAvailability will be migrated to use enum values: "none", "ready", "in_progress"
+// Enum values "blob" and "pool" will be deprecated soon.
 func buildDisplayName(displayName string, prebuildAvailability string) string {
 	prebuildText := ""
 
-	if prebuildAvailability == "blob" || prebuildAvailability == "pool" {
+	if prebuildAvailability == "blob" || prebuildAvailability == "pool" || prebuildAvailability == "ready" {
```

Comment:
Style nit: this whole function could be converted to a `switch` statement to promote readability:
```go
switch prebuildAvailability {
case "blob", "pool", "ready":
  return displayName + " (Prebuild ready)"
case "in_progress":
  return displayName + " (Prebuild in progress)"
default:
  return displayName
}
```